nzedb/NZBImport.php

```diff
@@ -297,15 +297,25 @@ protected function scanNZBFile(&$nzbXML, $useNzbName = false)
 			// Get the group names, groups_id, check if it's blacklisted.
 			$groupArr = [];
 			foreach ($file->groups->group as $group) {
-				$group = (string)$group;
+				$group = strtolower($group);
```

Comment:
We can't force to lowercase here. Uppercase letters are valid in group names.

Reply:
ok but the system does only accept one type. 
you can't have group "a.b.something" in datebase when nzb contains a group named "ab.SomeThing": import of this nzb fails. because it tries to create the group with capitals too which is not allowed. problem exists vice verse if you create group with capital letters like "test.MyGroup" for example and try to add another group "test.mygroup" you get a dead white page.

Reply:
ok, I'll look into that.